qt/python/mantidqt/gui_helper.py

```diff
@@ -18,6 +18,8 @@ def set_matplotlib_backend():
     This will set the backend if it hasn't been already. It also returns
     the name of the backend to be the name to be used for importing the
     correct matplotlib widgets.'''
+    if 'MPLBACKEND' in os.environ:
```

Comment:
Looking at this again perhaps we should move this under `backend = matplotlib.get_backend()` and then return backend? That way the caller can always guarantee a string return type?
Could we add a comment like "If the environment overrides the backend then we don't want to stamp on that" as a comment for the early exit?

Reply:
As in 
```
if backend == "MPLBACKEND"
```
?